### DISCUSSION TITLE
Portals in major cities

### DISCUSSION THREAD

**Discussion by jeyrella** [chromiecraft/chromiecraft#4553] [Suggestions]
Please please **please** reconsider removing the portals in major cities.
While it is appreciated it is not "Blizzlike", they are incredibly useful as a Quality of Life addition for characters that are choosing to level via questing - in particular for first toons that cannot afford mage portals everywhere.  This pushes players even further towards BG being even faster to level comparatively than the insane pace BG xp already is, and effectively penalises players new to the server compared to those that leveled before TBC - and they already have a hard wall ahead of them catching up!

Level 60+ toons will be sat with HS in Shattrath in any case so that does not detract from this equation.
They do not act as a "second HS" since there is no (very) quick way back to Shattrath from cities.

**Comment by Amrx2** [comment]
Hi jeyrella, this was always a planned part from day one once portals were available elsewhere. Also, you can get from any capital city to another within minutes in Wrath due to the way they redesigned transport between them, with Zeppelins, Boats, Portals and Trams all taking you between capital cities already. 

If you do not wish to wait for the transport to arrive you can always request a portal from a friendly mage nearby!

**Comment by jeyrella** [comment]
Your arguments don't really make sense;  if one can take boats to another city "in minutes", why where there portals implemented in the first place?
The reverse is true, you haven't given an argument as to _why_ they were planned to be removed.  Nor why they were even added.

"Nearby" friendly mages won't exist for new players, since a) they'll all be in Shattrath, and b) they are already demanding 10g for the trouble, which is utterly untenable for a new player.

Again, it was not answered that all this effectively penalises new players to the server compared to existing ones.

**Comment by Amrx2** [comment]
Jeyrella, this isn't an argument, this was always in our progression since the start of Chromiecraft when we put them there to begin with. It was what we announced also months ago would take place as well.

**Reply by jbockmon** [reply]
Respectfully, I disagree with your position on this. 

Just because you made a decision on the outset of this server, doesn't mean you have to hold onto that decision. Moreover, if you have new information, you should update your views in all things. That said, this server has a serious problem: it is populated with more endgame players than new players. This has an obvious negative impact on the new player experience and affects new players in lots of ways including not having others to level with, not having others to quest with, over inflated AH prices, high concentration of twinks in BG, etc... 

It would behoove this server to make it new player friendly, so that the population grows. Allowing portals in capital cities is one way for this to happen. Capitals are pretty much dead, as the bulk population is now centered in Shat, again, endgame focused content. In all honesty, you all may way to consider portals as just **one of many** new player enticements that make this server stand out. If this doesn't happen, the server pop will dwindle and die as your endgame population will eventually get tired of wotlk after it lands.

**Reply by heyitsbench** [reply]
> Just because you made a decision on the outset of this server, doesn't mean you have to hold onto that decision.

Wouldn't exactly be the first time decision changes like this have happened, as can be seen by the dramatic progression differences between Vanilla content and TBC content.

**Comment by trinitarius** [comment]
I think there shouldn't have been portals in major cities to begin with; it was a terrible choice to add them. It takes just a few minutes in 3.3.5a to get from one place to another as has already been stated by others.

The fact that there have been portals before makes players want to have them back, even though to me that is not a valid reason to add them back. Also it is not penalising new players; rather the addition of portals before made players quite lazy. Those that start out new on CC don't mind much that they have to use the tram or zeppelin but those that have used portals all this time will mind because they got used to just running to the portal room instead of using the normal way. 

New players are not negatively impacted by removed portals but rather that there are no people playing in lower level brackets. Replayability is something that is lacking at CC and should be adressed with ways to incentivise people to level alts or start on CC in the first place. 

Instead of asking for portals we should ask for adding incentives to level characters in lower level brackets and to encourage new players to join, such as: Improving the RAF system, adding challenge modes (ZhengPeiRu's module: https://github.com/ZhengPeiRu21/mod-challenge-modes), add rewards for people levelling alts, reward people that are actively grouping up during levelling, advertising the project specifically with the core idea of CC: slow progression.
